python/semantic_kernel/orchestration/sk_function_base.py

```diff
@@ -199,3 +199,10 @@ def set_ai_configuration(
             SKFunctionBase -- The function instance
         """
         pass
+
+    def __repr__(self):
+        """
+        Override how the classes are printed as strings so we don't include
+        the class attributes.
+        """
+        return f"{self.__class__.__name__}()"
```

Comment:
I think this is not solving the issue, and that is that those fields are not actually used, they are recreated by validation functions, so I would suggest moving these values into "semantic_kernel/utils/validation.py" as constants, so that if someone does want to use them they can be easily imported from there, and they are maintained once in that file and used by the validation methods in that file.

Reply:
Makes sense. Thank you for your feedback. I see now that the validation methods already have these regex consts. Going to re-arrange them in the validation file so that users can pull the 'consts' in, if required. And this allows the methods to still use them.